src/router.ts

```diff
@@ -86,13 +87,13 @@ export class Router {
         return;
       }
 
-      const currentPrice = getPythPrice(new PublicKey(tokenToSplMint(dip_deposit.splToken)));
-      const fractionOfYear = (Date.now() - dip_deposit.expirationMs) / 365 * 24 * 60 * 60 * 1_000;
-      const vol = THEO_VOL_MAP[dip_deposit.splToken] * (1.15 + Math.random() / 10);
-      const thresholdPrice = blackScholes(currentPrice, dip_deposit.strike / 1_000_000, fractionOfYear, vol, 0.01, 'call');
+      const currentPrice = await getPythPrice(new PublicKey(tokenToSplMint(dip_deposit.splToken)));
+      const fractionOfYear = (dip_deposit.expirationMs - Date.now() ) / (365 * 24 * 60 * 60 * 1_000);
+      const vol = THEO_VOL_MAP.get(dip_deposit.splToken) * (1 + volSpread + Math.random() * volSpread);
```

Comment:
We shouldnt need a random addition to the vol spread. And that will likely skew it more than we want.

Reply:
This was so that MMs cant just game the known backstop vol. They obviously still can but at 10 IV, so be it. Aren't we taking a fixed fee elsewhere?